modules/flowable-idm-spring/src/test/java/org/flowable/spring/test/authentication/SpringPasswordEncoderTest.java

```diff
@@ -71,8 +71,10 @@ public void testValidateSpringPasswordEncoder() {
 
         IdmEngineConfiguration defaultIdmEngineConfiguration = IdmEngines.getDefaultIdmEngine().getIdmEngineConfiguration();
 
-        assertTrue(defaultIdmEngineConfiguration.getPasswordEncoder() instanceof ClearTextPasswordEncoder);
-        assertTrue(passwordEncoder instanceof SpringEncoder);
+        assertThat(defaultIdmEngineConfiguration.getPasswordEncoder())
```

Comment:
I would say that this test was wrong before. When creating the `IdmEngine` with spring we should be getting the one which is used in spring when doing `IdmEngines.getDefaultIdmEngine()` and not create a new one